packages/cli-tools/bin/streamr-keys-generate.ts

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env node
+import '../src/logLevel'
+import { KeyPairIdentity } from '@streamr/sdk/dist/types/src/identity/KeyPairIdentity'
+import { identityConfig, validKeyTypeValues } from '@streamr/sdk'
+import { binaryToHex } from '@streamr/utils'
+import { createCommand, Options } from '../src/command'
+
+createCommand()
+    .description('generate a public/private key pair based on the --key-type option')
+    .option('--key-type [key-type]', `one of: [${validKeyTypeValues.join(', ')}]`)
+    .action(async (options: Options) => {
+        if (!options.keyType) {
+            console.error(`Error: Please provide --key-type [one of: ${validKeyTypeValues.join(', ')}]`)
+        } else {
+            const config = identityConfig[options.keyType]
+            if (!config) {
+                console.error(`Error: Invalid key type. Must be one of: ${validKeyTypeValues.join(', ')}.`)
+            }
+            const identity = config.generate() as KeyPairIdentity
+            console.info(`Public key: ${await identity.getUserIdString()}`)
```

Comment:
We could output the data as JSON so that it is easy to read programatically. See e.g. `stream create`, and `internal operator-create` commands:
https://github.com/streamr-dev/network/blob/aa0869c1bccf893d44ea4e836acefe5996e42227/packages/cli-tools/bin/streamr-internal-operator-create.ts#L32

Reply:
Ok fixed. The JSON format is a little harder to read in case of long keys, but arguably they are hard to read in any case, so I'd say it's not significantly worse.
<img width="1395" alt="Screenshot 2025-04-28 at 15 45 24" src="https://github.com/user-attachments/assets/26a72d7f-840a-426e-9e7a-7714161aa1f0" />